script/validators/source_validator.py

```diff
@@ -224,6 +259,11 @@ def validate_file(file_path):
         if not validator(file_path):
             file_status = False
 
+    relative_path = file_path.replace(PELOTON_DIR, '')
```

Comment:
One more thing: Can you use [`os.path.relpath`](https://docs.python.org/3/library/os.path.html#os.path.relpath) here instead of string replace?